bin/test-merges.js

```diff
@@ -141,7 +141,15 @@ function checkoutGitCommit(commitId) {
           reject(err2);
           return;
         }
-        resolve(commitId);
+
+        // Pass along the original commit message
+        git.show(["-s", "--format=%B"], (err3, data3) => {
+          if (err3) {
+            reject(err3);
+            return;
+          }
+          resolve([commitId, data3.trim()]);
```

Comment:
Please update the JSDoc at the beginning of this function, since we're now resolving with something different.
I'd also suggest adding a return here, just to match prevailing style in this file.